ts/Extensions/RangeSelector.ts

```diff
@@ -1294,7 +1383,7 @@ class RangeSelector {
         }
 
         if (time && useUTC) {
-            date += time.getTimezoneOffset(date) * 60 * 1000;
+            date += time.getTimezoneOffset(date);
```

Comment:
Is this change expected?

Reply:
Yes I wrote the original code recently, it was broken, `getTimezoneOffset` does the multiplication itself.

Reply:
Okay, thanks! 👍

Reply:
@khlieng